schemas/answ_action_response.fbs

```diff
@@ -0,0 +1,17 @@
+// A message for holding information on the (immediate) outcome of a command to (or state change of) a filewriter
+
+file_identifier "answ";
+
+enum ActionType : byte { StartJob = 0, SetStopTime, StopNow, HasStopped }
```

Comment:
"HasStopped" does not sound like an action, more like we are mixing in outcomes.

Reply:
I decided to re-use this schema for sending "has stopped" messages as well as it was easier and faster than creating yet another schema with associated (de)serialisers and logic. Of curse, I can change that if you prefer.